i18n/nl.json

```diff
@@ -4828,87 +4820,87 @@
   },
   {
     "id": "model.config.is_valid.max_users.app_error",
-    "translation": "Ongeldig maximaal aantal gebruikers per team bij team instellingen. Moet een getal groter dan 0 zijn."
+    "translation": "Ongeldig maximaal aantal gebruikers per team bij teaminstellingen. Moet een getal groter dan 0 zijn."
   },
   {
     "id": "model.config.is_valid.message_export.batch_size.app_error",
-    "translation": "Message export job BatchSize must be a positive integer"
+    "translation": "Berichteenxport BatchSize moet een positief geheel getal zijn"
   },
   {
     "id": "model.config.is_valid.message_export.daily_runtime.app_error",
-    "translation": "Message export job DailyRuntime must be a 24-hour time stamp in the form HH:MM."
+    "translation": "Berichtenexport DagelijkeUitvoertijd moet een tijdsaanduiding van 24 uur zijn in de notatie UU:MM."
   },
   {
     "id": "model.config.is_valid.message_export.enable.app_error",
-    "translation": "Message export job EnableExport setting must be either true or false"
+    "translation": "Bericht exporttaak EnableExport instelling moet waar of onwaar zijn"
   },
   {
     "id": "model.config.is_valid.message_export.export_from.app_error",
-    "translation": "Message export job ExportFromTimestamp must be a timestamp (expressed in seconds since unix epoch). Only messages sent after this timestamp will be exported."
+    "translation": "Bericht Export job ExportFromTimestamp moet een tijdsaanduiding zijn (uitgedrukt in seconden sinds unix-epoch). Alleen berichten verzonden die na deze tijdsaanduiding, worden geëxporteerd."
   },
   {
     "id": "model.config.is_valid.message_export.export_type.app_error",
-    "translation": "Message export job ExportFormat must be one of 'actiance', 'csv' or 'globalrelay'"
+    "translation": "Bericht Export taak ExportFormat moet of 'actiantie', of 'csv' of 'globalrelay' zijn"
   },
   {
     "id": "model.config.is_valid.message_export.global_relay.config_missing.app_error",
-    "translation": "Message export job ExportFormat is set to 'globalrelay', but GlobalRelaySettings are missing"
+    "translation": "Bericht Export taak ExportFormat is ingesteld op 'globalrelay', maar GlobalRelaySettings ontbreken"
   },
   {
     "id": "model.config.is_valid.message_export.global_relay.customer_type.app_error",
-    "translation": "Message export GlobalRelaySettings.CustomerType must be set to one of either 'A9' or 'A10'"
+    "translation": "Berich Export GlobalRelaySettings. CustomerType moet worden ingesteld op 'A9' of 'A10"
```

Comment:
```suggestion
    "translation": "Bericht Export GlobalRelaySettings. CustomerType moet worden ingesteld op 'A9' of 'A10"
```

Reply:
do please update this strings in the translation server for the next translation PR.

Reply:
I can always approve the suggestion but if it is not updated in the translation server, next PR will have the same issue